src/support/azext_support/custom.py

```diff
@@ -6,174 +6,276 @@
+            )
+
+    class SupportTicketsCreate(_CreateTicket.SupportTicketsCreate):
+
+        @property
+        def content(self):
+            from azure.cli.core.aaz import AAZUndefined
+
+            body = super().content
+            args = self.ctx.args
+            service_name = parse_support_area_path(
+                body["properties"]["problemClassificationId"]
+            )["service_name"]
+            body["properties"]["serviceId"] = (
+                "/providers/Microsoft.Support/services/{0}".format(service_name)
+            )
+            if args.start_time == AAZUndefined:
+                start_time = datetime.utcnow().strftime(("%Y-%m-%dT%H:%M:%SZ"))
+                body["properties"]["problemStartTime"] = start_time
+            return body
+
+
+class TicketCreateNoSubscription(_CreateTicketNoSubscription):
+
+    def _cli_arguments_loader(self):
+        args = super()._cli_arguments_loader()
+        return set_configured_defaults(args)
+
+    def pre_operations(self):
+        from azext_support._validators import (
+            _validate_tickets_create_no_subscription,
+        )
+        from azure.cli.core.aaz import AAZUndefined
+
+        super().pre_operations()
+        args = self.ctx.args
+        if args.technical_resource != AAZUndefined:
+            _validate_tickets_create_no_subscription(
+                self.cli_ctx,
+                str(args.problem_classification),
+                str(args.ticket_name),
+                str(args.technical_resource),
+            )
+        else:
+            _validate_tickets_create_no_subscription(
+                self.cli_ctx, str(args.problem_classification), str(args.ticket_name)
+            )
+
+    class SupportTicketsNoSubscriptionCreate(
+        _CreateTicketNoSubscription.SupportTicketsNoSubscriptionCreate
+    ):
+
+        @property
+        def content(self):
+            from azure.cli.core.aaz import AAZUndefined
+
+            body = super().content
+            args = self.ctx.args
+            service_name = parse_support_area_path(
+                body["properties"]["problemClassificationId"]
+            )["service_name"]
+            body["properties"]["serviceId"] = (
+                "/providers/Microsoft.Support/services/{0}".format(service_name)
+            )
+            if args.start_time == AAZUndefined:
+                start_time = datetime.utcnow().strftime(("%Y-%m-%dT%H:%M:%SZ"))
+                body["properties"]["problemStartTime"] = start_time
+            return body
+
+
+class TicketList(_List):
+
+    class SupportTicketsList(_List.SupportTicketsList):
+
+        @property
+        def query_parameters(self):
+            from azure.cli.core.aaz import AAZUndefined
+
+            parameters = super().query_parameters
+            args = self.ctx.args
+            if args.filter == AAZUndefined and args.pagination_limit == AAZUndefined:
```

Comment:
Adjust query arg should be done in rewrote `pre_operation` through customerized  value assignment. 
you can refer to code here: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/operations/dns.py#L829-L832

Reply:
Followings are the same